pkgs/build-support/setup-hooks/wrap-gapps-hook.sh

```diff
@@ -36,9 +36,12 @@ wrapGAppsHook() {
   done
 
   if [ -z "$dontWrapGApps" ]; then
-    for i in $prefix/bin/* $prefix/libexec/*; do
-      echo "Wrapping app $i"
-      wrapProgram "$i" "${gappsWrapperArgs[@]}"
+    # Loop over files, without falling prey to filenames containing
```

Comment:
Before, it was attempting to wrap all files in `$prefix/bin` and `$prefix/libexec`, including non-executable files and directories.  Additionally, it ignored files in subdirectories of these roots.

`emacs`, in particular, was broken because of the attempt to wrap directories.